src/main/java/ch/njol/skript/expressions/ExprHealth.java

```diff
@@ -19,6 +20,7 @@
 @Description("The health of a creature, e.g. a player, mob, villager, etc. The minimum value is 0, and the maximum is the creature's max health (e.g. 10 for players).")
 @Examples({"message \"You have %health% HP left.\""})
 @Since("1.0")
+@AvailableEvents(EntityDamageEvent.class)
 @Events("damage")
```

Comment:
Based on the commented-out code, I think you can instead remove the Events annotation here. It doesn't appear to have event-specific behavior anymore